0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -152,7 +152,12 @@ See below for the [ILP Error Format](#ilp-error-format) and [ILP Error Codes](#i
 
 ### ILP Payment Packet Format
 
-Here is a summary of the fields in the ILP payment packet format:
+There are two types of payment packets: quoted and forwarded. A quoted payment specifies the amount that should arrive at the destination, a forwarded payment doesn't.
```

Comment:
The difference is between whether it is forwarded or delivered, as per https://github.com/interledger/rfcs/issues/77. Whether you get a quote before sending the payment is a separate issue

Reply:
ok, renamed 'quoted' to 'delivered'.